src/index.ts

```diff
@@ -6,7 +6,8 @@ function arrayMustNotBeEmpty(value: Array<any>): boolean {
 }
 
 function emailMustBeValid(value: string): boolean {
```

Comment:
Maybe name it `stringMustBeValidEmail`, to emphasize the parameter must be a string ?
It will add consistency also.
Or else we could have multiple *email* rules and it would fit in an "email" category, not "string" anymore ?